.github/workflows/release-phar.yml

```
@@ -1,46 +1,53 @@
+  workflow_dispatch:
+  release:
+    types: [published]
 
 jobs:
   release:
     runs-on: ubuntu-latest
-
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)